susemanager-utils/susemanager-sls/susemanager-sls.spec

```diff
@@ -57,6 +57,8 @@ BuildRequires:  %{use_python}-salt
 BuildRequires:  python3-spacewalk-certs-tools
 # Different package names for SUSE and RHEL:
 Requires:       (python3-PyYAML >= 5.1 or python3-pyyaml >= 5.1)
+
+Requires:  %{use_python}-psycopg2
```

Comment:
don't know if you want to indent the new requires with the others.  Apart from this, thePR LGTM, only the bsc reference in the changes is  missing